### ISSUE TITLE
Pesterchum doesn't work, it doesn't even change my chumhandle after I enter it for the first time

### ISSUE THREAD

**Issue by ThePainkiller** [illuminatedwax/pesterchum#76]
The problem is the following: I've downloaded Pesterchum 3.41 from scratch, not the old version + the crack, the NEW version, and when I try to enter a chumhandle for the first time, I check the "set as default" box, and start the program. It says "CON3CT1NG". On the window it still says "MYCHUMHANDLE: pesterClient222" (or any randomized number).
I cannot change my chumhandle, I cannot even go Random Encounters, I try adding friends but they aren't even online to me. I cannot use the program at all, despite it (and calsprite) running perfectly fine. HELP!!

I'm running Windows XP, Service Pack 3, with almost every library available to make it function (Python? zlib? NET Framework?)

**Comment by illuminatedwax** [comment]
1) One of your profile files may be corrupted. How are you trying to change
your handle?
2) Make sure you aren't in "Low Bandwidth Mode" (in Options)

**Comment by ThePainkiller** [comment]
1) It is the first time I'm running Pesterchum in all of my life, there are no profiles saved, I can't save a profile. Like I said, I open the program, "CHANGING from pesterClientxyz", insert a random chumhandle, click "Set as default", click ok, I still can't chat with anyone.
I tried using "Switch", but I press the button and nothing happens.
Then I try the NickServ to "register" a nickname and it says this:

[13:16] PC: register metalApotheosis
[13:16] NS: Nickname pesterClient745 may not be registered.

2) Low Bandwith Mode is unchecked, so I'm not using it.

**Comment by adioztoreador** [comment]
hey hey I have that same problem! :C

**Comment by ThePainkiller** [comment]
No response yet, program still refuses to work....

**Comment by illuminatedwax** [comment]
Can you try installing and running the debug version? Basically run that,
then try to do the same thing (it will fail), and send me the contents of
debug.log?

**Comment by ThePainkiller** [comment]
Uninstalled Pesterchum. Re-downloaded and re-installed. Ran pesterchum_debug.exe, here's the log from the debug:

http://pastebin.com/Wec8ahJA

And here a screenshot of my pesterchum client with the random handle generated:

http://i.imgur.com/Yc5aGke.png

I tried to go into PROFILE -> SWITCH, but nothing happens.

I click Random encounter, nothing happens.

I think I might be missing a library or something. I hope this is good enough.
